src/examples/typography/CodeSize.tsx

```diff
@@ -10,20 +10,22 @@ import styled from 'styled-components';
 import { Code } from '@zendeskgarden/react-typography';
 
 const StyledDiv = styled.div`
-  margin-bottom: 12px;
+  margin-bottom: ${p => p.theme.space.sm};
 `;
 
 const Example = () => (
   <>
     <StyledDiv>
-      <Code size="small">Veggies es bonus vobis</Code>
+      <Code hue="grey" size="small">
+        Veggies es bonus vobis
+      </Code>
     </StyledDiv>
     <StyledDiv>
-      <Code size="medium" hue="green">
+      <Code hue="grey" size="medium">
         Veggies es bonus vobis
       </Code>
     </StyledDiv>
-    <Code size="large" hue="yellow">
+    <Code hue="grey" size="large">
```

Comment:
Don't add the default `hue="grey"` to this example. It was only needed in order to represent specific treatment in the color-based example.

Reply:
I've removed `hue=grey` here and added it to the color examples. Apologies for the confusion, the Google Doc example threw me off as it only showed three colors and I hadn't checked hues that were allowed.